infra/launch.sh

```diff
@@ -1,23 +0,0 @@
-#!/bin/bash
```

Comment:
can we leave this here for a bit just to give people a chance to migrate

Reply:
Good call, reverted.